jms/org.wso2.carbon.transport.jms/src/main/java/org/wso2/carbon/transport/jms/utils/JMSUtils.java

```diff
@@ -58,35 +55,7 @@ public static Destination lookupDestination(Context context, String destinationN
         if (null == destinationName) {
             return null;
         }
-        try {
             return JMSUtils.lookup(context, Destination.class, destinationName);
-        } catch (NameNotFoundException e) {
```

Comment:
This doesn't look good as we are removing exception handling. How this can solve the issue raised in https://github.com/ballerinalang/ballerina/issues/2044 ?